Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM golang:1.7.0
```

Comment:
should we go to 1.8 or 1.9?

Reply:
1.8 or 1.9 should be fine, but the jump is a little scary before cutting a release. I'll update this to select the current patch version of 1.7.